app/src/main/res/layout-land/add_profile_activity.xml

```diff
@@ -196,7 +196,7 @@
                 android:layout_height="wrap_content"
                 android:fontFamily="sans-serif"
                 android:text="@string/add_profile_allow_download_heading"
-                android:textColor="@{viewModel.validPin ? @color/oppiaPrimaryTextDark : @color/grey}"
+                android:textColor="@color/accessible_grey"
```

Comment:
I think in this PR we have to replace `grey` with `accessible_grey `. 

Example:
`android:textColor="@{viewModel.validPin ? @color/oppiaPrimaryTextDark : @color/accessible_grey}"`

Can you check if the scanner passes for both these cases of color code.

Reply:
|Original PR| After Suggested Changes |
|:-:|:-:|
|![Screenshot_1611641944](https://user-images.githubusercontent.com/64064110/105808011-031b7e80-5fcd-11eb-9b80-afdf5feb5894.png)|![Screenshot_1611642079](https://user-images.githubusercontent.com/64064110/105808017-057dd880-5fcd-11eb-8a5a-2f2ab54b26c0.png)|
|![Screenshot_1611642012](https://user-images.githubusercontent.com/64064110/105808015-04e54200-5fcd-11eb-8178-2a41176ec2be.png)|![Screenshot_1611642131](https://user-images.githubusercontent.com/64064110/105808018-06166f00-5fcd-11eb-8369-928e60293ee7.png)|

Scans are passing for both but Chantel's commented to keep the text color to be this for the entire time, but if this passes then it is better I guess , WDYT?

Reply:
I think we need this change. Lets merge this as this is more correct.

Reply:
The earlier implementation was actually more correct because when the checkbox is selected the text becomes highlighted which gives better look.

Reply:
I think you meant to say until the Text Input Layout have some content them, the textviews seemed to be inactive. Because checkbox controls the visibility of all the content below it.

Reply:
Yeah, you are right, the correctness of pin controls the color of switch related texts.